app/background.process.development.js

```diff
@@ -29,7 +32,7 @@ const manageLibLoading = ( store ) =>
     const listenerState = store.getState();
     const islistenerMock = listenerState.peruseApp.isMock;
 
-    if( islistenerMock === mockStatus )
+    if ( islistenerMock === mockStatus )
```

Comment:
@hunterlester do you think we should move `isMock` to the `authenticator` reducer? Does it make more sense there?

Reply:
Perhaps `isMock` should be in both `peruseApp` and `authenticator` states since each is associated with a respective native libarary?
Possibly useful if there were ever the case where `authenticator` was mock and `peruseApp` non-mock? Don't know.

Reply:
Hmmm, I dont think that can happen, or? It's the auth that decides the network state not the app (which could auth against it in non-dev mode right?)